templates/README.toplevel

```diff
@@ -0,0 +1,6 @@
+The subdirectories here each contain a build of the KG-COVID-19 knowledge graph.
+The directories are named with a timestamp (YYYYMMDD) according to when the build
```

Comment:
`timestamp` should be replaced. i.e. `...with a date (ISO 8601 format) according to when the build...`

Reply:
Okay, thanks - done